### PR TITLE
refactor: Svn errors now show the repo name in the output channel

### LINKED ISSUE #513: Get rid of svn cat warnings
```
[trunk]$ svn cat /home/cjohnston/dev/test/trunk/src/include/classes/base/ical_dataclass.php
svn: warning: W200005: '/home/cjohnston/dev/test/trunk/src/include/classes/base/ical_dataclass.php' is not under version control
svn: E200009: Could not cat all targets because some targets are not versioned
svn: E200009: Illegal target for the requested operation
```

### PR DESCRIPTION
**Pull request by JohnstonCode**
fixes #513

### CODE REVIEW

**Comment by JohnstonCode** [comment]
@edgardmessias review?

**Comment by edgardmessias** [comment]
Yes, you can chage to:
```ts
    if (options.log !== false && stderr.length > 0) {
      const name = this.lastCwd.split(/[\\\/]+/).pop();
      const err = stderr
        .split("\n")
        .filter((line: string) => line)
        .map(
          (line: string) => `[${name}]$ ${line}`
        )
        .join("\n");
      this.logOutput(err);
    }
```

**Comment by JohnstonCode** [comment]
Yeah i did think about doing all the name stuff in the map. good suggestion

File: src/svn.ts [hunk 1]
```diff
@@ -201,7 +203,14 @@ export class Svn {
     const decodedStdout = iconv.decode(stdout, encoding);
 
     if (options.log !== false && stderr.length > 0) {
-      this.logOutput(`${stderr}\n`);
+      const err = stderr
+        .split("\n")
+        .filter((line: string) => line)
+        .map(
+          (line: string) => `[${this.lastCwd.split(/[\\\/]+/).pop()}]$ ${line}`
```
[the remark this answers is not in the document]
Comment (JohnstonCode) [reply src/svn.ts]: Done

**Pull request merged**